src/batchify.js

```diff
@@ -0,0 +1,88 @@
+/**
+ * @license Copyright (c) 2003-2015, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+'use strict';
+
+/**
+ * Creates set of operations which need to be applied to the input in order to transform
+ * it to the output. Can be used with strings or arrays.
+ *
+ *		const input = Array.from( 'abc' );
+ *		const output = Array.from( 'xaby' );
+ *		const batch = batchify( diff( input, output ), output );
+ *
+ *		batch.forEach( operation => {
```

Comment:
In similar way you have model in the UI and model in the engine. The same name for things which are significantly different. 

If you're able to find a name which still makes sense, then I'm ok with it. But I don't want to change the name to some weirdo just to avoid collision with totally different package. Certain concepts may repeat and trying to find a new name every time you implement them is pointless.